index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel= "https://fonts.googleapis.com/css2?family=Inter&display=swap" href="style.css">
+  <title> My portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="container">
+      <ul>
+        <li><a><h2 class="upitem">Welcome </h2></a></li>
+        <li><img src="./imges/Union.svg" alt="menu"></li>
```

Comment:
- [ ] Hey @cilfonegabriel  what do you think if we wrap your menu icon inside an `a` tap?

Reply:
Fixed.